packages/docs/pages/Collapse/CollapsePage.tsx

```diff
@@ -0,0 +1,40 @@
+import { Collapse, H0, H1, Text } from '@bigcommerce/big-design';
+import React from 'react';
+
+import { Code, CodePreview } from '../../components';
+import { CollapsePropTable } from '../../PropTables';
+
+export default () => (
+  <>
+    <H0>Collapse</H0>
+
+    <Text>
+      A <Code primary>Collapse</Code> component allows to show/hide a piece of data which is not relevant for users in a
+      majority of use cases in order to save screen space.
+    </Text>
```

Comment:
```suggestion
    <Text>
      A <Code primary>Collapse</Code> component allows for showing/hiding content.
    </Text>
```

Reply:
The previous version was from the description in Figma.